### ISSUE TITLE
Extension button no longer working in Chrome 

### ISSUE THREAD

**Issue by Stealinglight** [rshev/MoveTabToNextWindow#15]
Starting to day my button isn't working anymore in Brave ([Version 1.49.132 Chromium: 111.0.5563.147 (Official Build) (arm64)])
When clicking the button there's no action, when I right click and go to move to next window it brings me to the github.

**Comment by rshev** [comment]
Hi, checking Brave 1.52.129 (114.0.5735.198) and the extension seems to be working. Anything you see in the console if you inspect via extensions -> inspect views on it?

**Comment by eggbean** [comment]
I deleted my earlier comment as I thought it wasn't working, but it has started working again. It did since then stop working once again, but it's working at the moment. I may have something to do with another extension I use. (Using Chrome)

If it stops working again I'll look in the console.

**Comment by eggbean** [comment]
I haven't tried using this extension for a while, but I have been trying tonight and it's not working again. I don't know what I am looking for in the console, but this is a screenshot of it for this webpage:

![image](https://github.com/rshev/MoveTabToNextWindow/assets/815311/527537ea-5705-44fe-bab0-978961efdf3b)


I have quite a few other extensions, but I have also tried after disabling some of them.

If I click on the the "top" dropdown menu at the top of the console, I can see the names of some other extensions, but not this one, but the icon for the extension is visible in the toolbar.

**Comment by dimateos** [comment]
Maybe related to https://github.com/rshev/MoveTabToNextWindow/issues/4, having an installed web app open in its own window while using this seems to break itsd funcionality, even after closing them (corrupted tab locations etc?)

**Comment by rshev** [comment]
Good point, I don't use web apps, need to exclude their windows. Will take a look over the weekend, thanks for the tip!

**Comment by rshev** [comment]
Addressed in b0e09b32ea63bb409d9c3d9f72095e81df5ae786 and uploaded version 1.2.0 to the stores. Thank you for your patience.
